drivers/ethernet/dsa_ksz8xxx.c

```diff
@@ -137,7 +152,7 @@ static int dsa_ksz8xxx_probe(struct ksz8xxx_data *pdev)
 	/*
 	 * Wait for SPI of KSZ8794 being fully operational - up to 10 ms
```

Comment:
need to update the comment? since it looks like now its waiting up to 1 ms?

Reply:
I returned the timeout in the code to the original driver's 10ms, as this may be a specific value for the ksz8794 chip.